ui/apps/platform/src/Containers/NetworkGraph/flows/advancedFlowsFilterUtils.ts

```diff
@@ -0,0 +1,42 @@
+import { AdvancedFlowsFilterType, FilterValue } from './types';
+
+function isValidPort(port: string): boolean {
+    const portNum = parseInt(port, 10);
+    if (Number.isNaN(portNum)) {
+        return false;
+    }
+    return portNum >= 1 && portNum <= 65535;
```

Comment:
just curious, where are you getting these numbers? is it possible to get non-valid port numbers from the API resopnse?

Reply:
I'm assuming we will always get valid ports, but this is more to distinguish what is a port value in that `selections` array